src/tests/unit/tests/electron/platform/android/setup/live-android-setup-deps.test.ts

```diff
@@ -37,12 +41,18 @@ describe('LiveAndroidSetupDeps', () => {
         );
         fetchConfigMock = Mock.ofInstance((port: number) => new Promise<DeviceConfig>(() => null));
         loggerMock = Mock.ofType<Logger>();
+        adbWrapperFactoryMock = Mock.ofType<AdbWrapperFactory>();
+        adbWrapperHolderMock = Mock.ofType<AdbWrapperHolder>();
```

Comment:
Can these be strict?
```suggestion
        adbWrapperFactoryMock = Mock.ofType<AdbWrapperFactory>(undefined, MockBehavior.Strict);
        adbWrapperHolderMock = Mock.ofType<AdbWrapperHolder>(undefined, MockBehavior.Strict);
```